exercises/G-conditionals-2/exercise-3.js

```diff
@@ -8,7 +8,17 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+function calculateGrade(mark) {
```

Comment:
Great job Ekrem.

You could also do this in the reverse order using `>=` but the results would be the same. I like what you did here.